clang-tools-extra/docs/clang-tidy/checks/cppcoreguidelines/avoid-goto.rst

```diff
@@ -50,3 +50,12 @@ Modern C++ needs ``goto`` only to jump out of nested loops.
   some_operation();
 
 All other uses of ``goto`` are diagnosed in `C++`.
+
+
+Options
+-------
+
+.. option:: IgnoreMacros
+
+   If set to `true`, the check will not warn if ``goto`` statement is placed
```

Comment:
I think 'expanded from a macro ' sounds better than 'placed'

---

'if a go-to statement'